lib/Markup/Markup.cpp

```diff
@@ -15,6 +15,9 @@
 #include "swift/AST/Comment.h"
 #include "swift/Markup/LineList.h"
 #include "swift/Markup/Markup.h"
+#if defined(__CYGWIN__)
```

Comment:
We have our own fork of CMark; please submit this as a patch there (or better yet submit it upstream).